po/de.po

```diff
@@ -374,7 +363,7 @@ msgstr "Konnte Datei '%ls' nicht zu '%ls' umbenennen: %s"
 #: src/env_universal_common.cpp:485
 #, c-format
 msgid "Unable to open temporary file '%ls': %s"
-msgstr ""
+msgstr "Die temporäre Date '%ls' kann nicht geöffnet werden: %s"
```

Comment:
Typo: "Datei" (or is this austrian?)

Reply:
It's just me not hitting the "i" key hard enough

Reply:
It's just me not hitting the "i" key hard enough